### ISSUE TITLE
toml files should use quotes

### ISSUE THREAD

**Issue by wildmolasses** [Great-Hill-Corporation/quickBlocks#1000]
Our dappnode project has us reading toml files not just in C++ but in javascript as well. Many toml parsers expect quotes ( examples: https://toml-to-json.matiaskorhonen.fi/ and https://www.tomllint.com/ ) and will error if they don't get them. From the [toml spec](https://github.com/toml-lang/toml#string), 

> Basic strings are surrounded by quotation marks. 
...
> Values must be of the following types: String, Integer, Float, Boolean, Datetime, Array, or Inline Table. Unspecified values are invalid.

How to resolve: change the default toml files in /src/other/install to use quotes. Review the chifra tool and correct toml handling if anything's wrong. Run tests. Changing the toml to use quotes could possibly break certain parts of the code. A careful review is required.

**Comment by tjayrush** [comment]
It won't break anything if there are quotes (as opposed to no quotes). It actually expects quoted strings, but it accepts unquoted ones. This is a larger issue (isn't everything), documented here: #583. I delayed it until the next release.

I also made the adjustments to the ./src/other/install/*.toml files and the local ~/.quickBlocks/*.toml files and re-ran the tests and everything still passes.

I'll delay this one and re-open when we get to the next release. Thanks.

I removed 'Good first issues' because it is totally not a good first issue. It's a mess.
